### PR TITLE
Add yaml file with correspondance between Zenodo ext and Galaxy datatype

### PR DESCRIPTION
**Pull request by bebatut**
It is needed for https://github.com/galaxyproject/planemo/pull/861

### CODE REVIEW

File: shared/datatypes.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+# Keep the alphabetical order
+bam: bam
+bed: bed
+bedGraph: bedgraph
+counts: tabular
+csv: csv
+design: mothur.design
+fastq: fastqsanger
+fastqsanger: fastqsanger
+fastqsanger.gz: fastqsanger.gz
```
Comment (bgruening) [inline shared/datatypes.yaml]: fastqsanger.bz

File: shared/datatypes.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+---
+# This file makes the correspondance between extensions on Zenodo (first element) and Galaxy datatypes (second element)
+# Keep the alphabetical order
+bam: bam
+bed: bed
+bedGraph: bedgraph
```
Comment (bgruening) [inline shared/datatypes.yaml]: bg

File: shared/datatypes.yaml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+counts: tabular
+csv: csv
+design: mothur.design
+fastq: fastqsanger
+fastqsanger: fastqsanger
+fastqsanger.gz: fastqsanger.gz
+fna: fasta
+fq: fastqsanger
+gbk: gbk
+gff: gff
```
Comment (bgruening) [inline shared/datatypes.yaml]: gff3 maybe as well?

File: shared/datatypes.yaml [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+gtf: gtf
+gz: gz
+metadata: tabular
+mgf: ''
+mzml: ''
+raw: ''
+sqlite: ''
```
Comment (bgruening) [inline shared/datatypes.yaml]: why is this empty?

File: shared/datatypes.yaml (hunk 2 above)
Comment (bebatut) [reply shared/datatypes.yaml]: So you have your own format :smile:

Comment (bebatut) [reply shared/datatypes.yaml]: Joking aside, is it bedgraph?

Comment (bgruening) [reply shared/datatypes.yaml]: Yes its a pretty common ext for bedgraph.

**Pull request merged**